pyautomagic/preprocessing/performEOGRegression.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+
+
+def performEOGRegression(eeg, eog):
+
+    """clean_eeg = performEOGRegression(eeg,eog)
+        performs linear regression to remove eog artifacts from eeg data
+        INPUT: eeg, eog data matrices
+        OUTPUT: clean_eeg matrix """
+
+    size_eeg = np.shape(eeg)
+    size_eog = np.shape(eog)
+    if size_eeg == 0 or size_eeg == 0:  # check if the inputs values for eeg and eog are null
```

Comment:
You should have an error raised here with a description of what they need to fix.